src/domains/dashboard/pages/Dashboard/Dashboard.tsx

```diff
@@ -23,6 +23,7 @@ export const Dashboard = ({ networks, portfolioPercentages, balances }: Dashboar
 	const [showTransactions, setShowTransactions] = useState(true);
 	const [showPortfolio, setShowPortfolio] = useState(true);
 	const [allTransactions, setAllTransactions] = useState<Contracts.TransactionDataType[] | undefined>(undefined);
+	const [loadingTransactions, setLoadingTransactions] = useState(true);
```

Comment:
Nice work @goga-m! 🥇 

One minor, I think that we can use `isLoadingTransactions` just to keep the pattern since the property that we are using in the transactions component is `isLoading` otherwise such a fine work 👌 


Well done! 👯‍♀️ 

```suggestion
	const [isLoadingTransactions, setIsLoadingTransactions] = useState(true);
```

Reply:
Thanks @clucasalcantara for taking the time to review. You are absolutely right :+1: